code/glTFAsset.h

```diff
@@ -786,8 +790,10 @@ namespace glTF
         void AttachToDocument(Document& doc);
         void DetachFromDocument();
 
+#ifdef  GLTF_ASSET_WITH_EXPORT_SUPPORT
```

Comment:
Makes sense. The more obvious fix is then to include the exporter implementation headers in all cases. I'll do that.